common/templates/navbar.html

```diff
@@ -40,7 +49,10 @@
                         </ul>
                     </li>
                 </ul>
-            </div><!--/.nav-collapse -->
+            </div><!--/.nav-collapse -->    
         </div>
     </nav>
 </header>
+
+<!--Modal Window -->
+<div profilemodal id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" class="modal fade" ></div>
```

Comment:
Why does this modal element need to be defined here? 

In all other cases we use the modal popup, the modal is a template that is rendered into the DOM with only having to specify it as part of the `$uibModal.open` function.

Reply:
In my local code, removing this element from the DOM doesn't appear to change any of the functionality.